txpipe/twopoint_fourier.py

```diff
@@ -540,12 +541,15 @@ def compute_power_spectra(self, pixel_scheme, i, j, k, maps, workspaces, ell_bin
         workspace = workspaces[(i,j,k)]
 
         # Get the coupled noise C_ell values to give to the master algorithm
-        cl_noise = self.compute_noise(i, j, k, ell_bins, maps, workspace)
+        if self.config['true_shear']:
```

Comment:
Isn't this the wrong way around?

You could also put it inside the compute_noise function to make the top level look simpler.

Reply:
Yes, thanks for catching this, I just changed the option at the last moment and forgot to switch the boolean.
I also put the it inside the compute_noise function.